### PR TITLE
Int news

### PR DESCRIPTION
**Pull request by romaoksenyuk**
Додати Rest API ендпойнти для int_news
https://trello.com/c/6JTuRbKt/72-%D0%B4%D0%BE%D0%B4%D0%B0%D1%82%D0%B8-rest-api-%D0%B5%D0%BD%D0%B4%D0%BF%D0%BE%D0%B9%D0%BD%D1%82%D0%B8-%D0%B4%D0%BB%D1%8F-intnews

### CODE REVIEW

**Comment by klekhaav** [comment]
please resolve conflicts

File: api/ext_news/views.py [hunk 1]
```diff
@@ -34,11 +34,11 @@ class MailingAPIView(APIView):
     def get(self, request, **extra_kwargs):
         queryset = self.queryset
         for user in queryset.iterator():
-            News.mailing(data=user)  
-        return Response({'Status':'OK'},status=status.HTTP_200_OK)
+            News.mailing(data=user)
+            return Response({'Status':'OK'},status=status.HTTP_200_OK)
```
Comment (pavlo-mk) [inline api/ext_news/views.py]: You get return method for first item in `for` cycle. Is it correct?

File: api/ext_news/views.py [hunk 2]
```diff
@@ -34,11 +34,11 @@ class MailingAPIView(APIView):
...
-def Mailing():
-    queryset = StdUser.objects.filter(**{'news_subscription':True})
-    for user in queryset.iterator():
-        News.mailing(data=user)
-    return None
+    def Mailing():
+        queryset = StdUser.objects.filter(**{'news_subscription':True})
+        for user in queryset.iterator():
+            News.mailing(data=user)
+            return None
```
Comment (pavlo-mk) [inline api/ext_news/views.py]: the same as above. get return method for first item in `for`.

File: api/ext_news/views.py [hunk 3]
```diff
@@ -34,11 +34,11 @@ class MailingAPIView(APIView):
...
-        return Response({'Status':'OK'},status=status.HTTP_200_OK)
+            News.mailing(data=user)
+            return Response({'Status':'OK'},status=status.HTTP_200_OK)
 
-def Mailing():
-    queryset = StdUser.objects.filter(**{'news_subscription':True})
-    for user in queryset.iterator():
-        News.mailing(data=user)
-    return None
+    def Mailing():
```
Comment (pavlo-mk) [inline api/ext_news/views.py]: please use name for methods with small letters.

**Review by pavlo-mk — CHANGES_REQUESTED** [review]
Requested changes in this PR. I didnt uptedstand why you added all files in this PR, whats happened in your editor?)🤔

**Pull request merged**